test/unit/whenKeyTest.js

```diff
@@ -2,9 +2,20 @@ const test = require('tape');
 const helpers = require('../helpers/createDanceAPI');
 
 test('make sure we are checking all relevant keys', async t => {
-  const nativeAPI = await helpers.createDanceAPI();
+  let lastEvents;
+  const nativeAPI = await helpers.createDanceAPI({ onHandleEvents: events => {
+    lastEvents = events;
+  }});
+
   const p5 = nativeAPI.p5_;
 
+  nativeAPI.play({
```

Comment:
Should we `await` this `nativeAPI.play()` call like you did elsewhere in tests?

Reply:
No, `play()` no longer returns a `Promise` after your feedback. I probably need to remove the `await` in a few places now.